server/impl/DealApiImpl.cpp

```diff
@@ -69,7 +69,7 @@ void DealApiImpl::get_deal(const Pistache::Optional<std::string> &seed, const Pi
       ww_deal_to_uuid(deal_uid, uuid);
       deals.push_back(std::string(uuid));
    }
-
+   response.headers().add<Http::Header::AccessControlAllowOrigin>("*"); // ACCESS CONTROL ALLOW ORIGIN (CORS)
```

Comment:
Moving the comment from the end line to the previous line would help reduce line length.